arch/arm64/boot/dts/qcom/msm8953-xiaomi-tissot.dts

```diff
@@ -47,6 +73,11 @@
 			no-map;
 		};
 
+		cont_splash_mem: cont-splash@90001000 {
```

Comment:
this requires a
`
/delete-node/ &cont_splash_mem;
`
on top of this file

Reply:
true. I didn't notice that this region was added to msm8953